js/main.js

```diff
@@ -0,0 +1,14 @@
+function randomInteger(min,max){
+  if(min>=0 && max>=0 && min<max){
```

Comment:
Операторы от операндов лучше отделять пробелом, также и фигурную скобку после условия
```suggestion
  if(min >= 0 && max >= 0 && min < max) {
```